packages/app-builder-lib/src/util/appFileCopier.ts

```diff
@@ -18,26 +18,73 @@ const BOWER_COMPONENTS_PATTERN = `${path.sep}bower_components${path.sep}`
 /** @internal */
 export const ELECTRON_COMPILE_SHIM_FILENAME = "__shim.js"
 
-export function getDestinationPath(file: string, fileSet: ResolvedFileSet) {
-  if (file === fileSet.src) {
+function removePnpmAndNextTwoFolders(file: string) {
+  // Split the path into parts
+  const parts = file.split(path.sep)
+
+  // Find the index of the '.pnpm' folder
+  const pnpmIndex = parts.findIndex(part => part === ".pnpm")
+
+  // If '.pnpm' is found, and there are at least two more folders after it
+  if (pnpmIndex >= 0 && parts.length > pnpmIndex + 2) {
+    // Remove '.pnpm' and the next two folders from the parts array
+    parts.splice(pnpmIndex, 3)
+  }
```

Comment:
Are we able to make this guarantee that the two nested folders within `.pnpm` folder are okay to be removed? Are they unnecessary symlinks or something?